### PR TITLE
#46 return user data

### PR DESCRIPTION
**Pull request by martenjostmann**

### CODE REVIEW

File: backend/users/views.py [hunk 1]
```diff
@@ -319,3 +321,24 @@ def get_balance(request):
...
         return Response("No card_id in user profile",
                         status=status.HTTP_404_NOT_FOUND)
+
+
+@api_view(['GET'])
+@permission_classes((permissions.IsAuthenticated,))
+def get_user_data(request):
+    """Get infos about the user
+
+        Route: api/v1/user/get_balance
```
[the remark this answers is not in the document]
Comment (martenjostmann) [reply backend/users/views.py:331]: Why I forget to rename that shit every time :D

[the remark this answers is not in the document]
Comment (erikzimmermann) [reply backend/users/views.py]: Yeah 😅

**Review by erikzimmermann — APPROVED** [review]

**Comment by lgiesen** [comment]
You could think about defining a first_name & last_name, but name is fine 👍

**Review by lgiesen — COMMENTED** [review]
You could think about defining a first_name & last_name, but name is fine 👍

**Comment by martenjostmann** [comment]
> You could think about defining a first_name & last_name, but name is fine 👍 

Yeah, maybe we can get the full name from the Learnweb. But up to now this is not important.

**Pull request merged**